wildlifelicensing/apps/returns/views.py

```diff
@@ -98,7 +102,13 @@ def _set_submitted(self, ret):
         if is_officer(self.request.user):
             ret.proxy_customer = self.request.user
 
-        ret.status = 'submitted'
+        # assume that all the amendment requests has been solved.
+        pending_amendments = ret.pending_amendments_qs
```

Comment:
Is it definitely the case the all amendments have been solved? Can user lodge a return if there are oustanding amendments?